packages/backend-common/src/cache/CacheManager.ts

```diff
@@ -44,7 +43,6 @@ export class CacheManager {
     redis: this.getRedisClient,
     memcache: this.getMemcacheClient,
     memory: this.getMemoryClient,
-    none: this.getNoneClient,
```

Comment:
We should keep this option.

Reply:
I'm curious why you want to keep it — if we want to support a no-cache option, wouldn't it be better to have the cache as TS-optional in `CacheManager.getClient` such that plugin authors can be aware when caching is not supported?

Suppose that a plugin author believes they're caching an API response since a cache client is defined, but they never find a cache entry and therefore accidentally pound the API aggressively. The fix for that is to then implement your own TTL in code, which should be the job of a cache implementation.

Reply:
Well it's not really the plugin author pounding the API at that point I feel, it's the integrator that disabled caching in their instance. It's a fair question why you would ever want to use this option in practice though, main reason I could think of is E2E testing.